backend/package.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "icard-backend",
```

Comment:
change all "icard" and "iCard" occurence into "ICard"

Reply:
I cannot change the name to ICard because `NPM packages are not allowed upper case characters in their name`. I can name it to icard if you want.